### PR TITLE
2016 mozfest session proposals

### PR DESCRIPTION
**Pull request by cadecairos**
@alicoding R?

### CODE REVIEW

**Comment by cadecairos** [comment]
cc/ @mmmavis

File: worker/mozfest_session_proposal_2016.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  var SAWMILL_EVENT = "mozfest_session_proposal_2016"
+
+  return function(event, cb) {
+    if (event.event_type !== SAWMILL_EVENT) {
+      return process.nextTick(cb);
+    }
+    var mail = mailroom.render(SAWMILL_EVENT, {
+      first_name: event.data.first_name,
+      github_issue_url: event.data.github_issue_url,
+      github_issue_title: event.data.github_issue_title
```
Comment (cadecairos) [inline worker/mozfest_session_proposal_2016.js]: @mmmavis this variable and the three above it (as well as the `SAWMILL_EVENT` variable) should give you all the info you need to set up the hook for sending emails on the mozfest server

**Comment by alicoding** [comment]
r+

File: worker/mozfest_session_proposal_2016.js (hunk 1 above)
Comment (mmmavis) [inline worker/mozfest_session_proposal_2016.js]: :bowtie:

**Pull request merged**